README.md

```diff
@@ -543,6 +543,7 @@ The following options are available globally and apply when going into global ed
 | pinning                   | `false` | Pin shared vertices/markers together during edit ⭐. [Details](#pinning)                                                     |
 | allowSelfIntersection     | `true`  | Allow/Disallow self-intersections on polygons and polylines.                                                                |
 | preventMarkerRemoval      | `false` | Disable the removal of markers/vertexes via right click.                                                                    |
+| preventObjectDelete       | `false` | Enable to disallow preventMarkerRemoval to remove the object. Requires preventMarkerRemoval: false                          |
```

Comment:
Let's rename this

Option Name: `removeLayerBelowMinVertexCount`
Default: true
Description: 
```
If `true`, vertex removal that cause a layer to fall below their minimum required vertices will remove the entire layer. If `false`, these vertices can't be removed. Minimum vertices are 2 for Lines and 3 for Polygons.
```

This requires a small change to your code to check for `removeLayerBelowMinVertexCount` to be `true` instead.